lib/miq_automation_engine/models/miq_ae_yaml_export.rb

```diff
@@ -102,7 +104,8 @@ def write_namespace_file(ns_obj)
   end
 
   def write_all_domains
-    MiqAeDomain.all.each do |dom_obj|
+    domains = @tenant ? @tenant.ae_domains : MiqAeDomain.all
+    domains.each do |dom_obj|
       @domain = dom_obj.name
       @domain_object = domain_object
       write_domain
```

Comment:
probably a future edit:

I think this should read: `write_domain(dom_obj)`
And is this even necessary? `@domain_object = domain_object`